### PR TITLE
Add Makefile To Build With Oct 2020 Emscripten

### PR DESCRIPTION
**Pull request by AE1020**
The previous build instructions would not work under the current
Emscripten.  They have moved away from offering LLVM BitCode as
a target (at least, there's no obvious way to get it).

If the `-c` option is used, then the generated .wasm file will be
relocatable and can act as an object file would.  However, this
does not allow the combining of multiple source files into a single
bitcode file in the way the instructions previously had.

Also: the -lGLU option does not seem to work in this version.  The
removal of the switch resulted in a successful link and run of the
built product.

This commit is an offering of the makefile which I used to build.
Doing better would involve putting the intermediate .wasm files in
their own directories etc.  But this was just to get it built for
a quick test, so improvements are left as an exercise to the
reader.  :-)

### CODE REVIEW

**Comment by AE1020** [comment]
Hi there, I thought this might be useful.

There is a thread making it [sound like they support bitcode](https://github.com/emscripten-core/emscripten/issues/10142#issuecomment-572294442) if you use `-flto` or `-s WASM_OBJECT_FILES=0` but that didn't work for me.

They say you can just use the .wasm files as objects, but that only seems to work if you use `-c` or it complains they are not relocatable.  There's `-s RELOCATABLE=1` but that doesn't seem to be applicable to this definition of "relocatable".  So it seems like you *have* to use `-c`, but if you do then it will only compile one file per command line.

So basically the only way I managed to get it to work was to use `-c` and that meant each file compiling individually.  Maybe there's another way, but this worked.

**Comment by danilw** [comment]
sorry for the late response

I do not "support" this code, it was one of my first "try of using WASM" 

better or make your own repository for "current" emscripten with build process you need

**Comment by AE1020** [comment]
> I do not "support" this code, it was one of my first "try of using WASM"

Not wanting to support something is fine.  Although I feel that since I went through the work of updating it, it would be nice if you took the PR.  Because your repository is found by Google searches (e.g. by a post you made on reddit or whatever).

If you are *not* going to accept PRs, then you should [Archive the Repository](https://github.blog/2017-11-08-archiving-repositories/).  That will mark it read only, and signal to people it may be out of date...and know ahead of time that any effort to work on it would be divergent.

If you don't want to merge this PR before archiving (for whatever reason) you could link to it from the README.md, along with a statement you are no longer involved...and that people who want to see if it works might start by looking here.

**Comment by danilw** [comment]
> If you are not going to accept PRs, then you should Archive the Repository.

yes I do it now, thanks

> If you don't want to merge this PR before archiving (for whatever reason) you could link to it from the README.md

il add line to inform people to look on other repository and that this repository is not updated anymore. Also I link your repo.
